docs/src/pages/components/popper/FakedReferencePopper.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Popper from '@material-ui/core/Popper';
+import Typography from '@material-ui/core/Typography';
+import Fade from '@material-ui/core/Fade';
+import Paper from '@material-ui/core/Paper';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    typography: {
+      padding: theme.spacing(2),
+    },
+  }),
+);
+
+interface AnchorType {
+  clientWidth: number;
+  clientHeight: number;
+  getBoundingClientRect(): ClientRect | DOMRect;
+}
+
+export default function FakedReferencePopper() {
+  const [open, setOpen] = React.useState(false);
+  const [anchorEl, setAnchorEl] = React.useState<AnchorType | null>(null);
```

Comment:
What do you think about
```suggestion
  const [anchorEl, setAnchorEl] = React.useState<PopperProps['anchorEl'] | null>(null);
```
? 

Helps IMO with code completion if later new optional properties are added. And it will also include potential documentation of `ReferenceObject` properties.

Reply:
Agree, that's much better